test/fixtures/transactions_data.js

```diff
@@ -0,0 +1,175 @@
+export default [
```

Comment:
@eniolam1000752 I believe filename should be renamed to camel case for consistency, preferably `transactions.js`